packages/pricing/jest.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  moduleNameMapper: {
+    "^@models": "<rootDir>/src/models",
+    "^@services": "<rootDir>/src/services",
+    "^@repositories": "<rootDir>/src/repositories",
+  },
+  globals: {
```

Comment:
Do you mind to adapt the jest.config like this [PR](https://github.com/medusajs/medusa/pull/4870/files) and upgrade typescript and jest version?
If that break something I can change it later